src/Utilities/Timer.f90

```diff
@@ -12,183 +13,171 @@ module TimerModule
+    !
+    ! -- Add or subtract seconds
+    elsec = elsec + (IEDT(5) - ibdt(5)) * 3600.0
+    elsec = elsec + (IEDT(6) - ibdt(6)) * 60.0
+    elsec = elsec + (IEDT(7) - ibdt(7))
+    elsec = elsec + (IEDT(8) - ibdt(8)) * 0.001
+    !
+    ! -- Convert seconds to days, hours, minutes, and seconds
+    ndays = int(elsec / nspd)
+    rsecs = mod(elsec, DSECPERDY)
+    nhours = int(rsecs / 3600.0)
+    rsecs = mod(rsecs, DSECPERHR)
+    nmins = int(rsecs / 60.0)
+    rsecs = mod(rsecs, DSIXTY)
+    nsecs = int(rsecs)
+    rsecs = mod(rsecs, DONE)
+    msecs = nint(rsecs * 1000.0)
+    nrsecs = nsecs
+    if (rsecs > 0.5) nrsecs = nrsecs + 1
+    !
+    ! -- Write elapsed time to screen
+    if (ndays > 0) then
+      write (line, fmttma) ndays, nhours, nmins, nrsecs
+    elseif (nhours > 0) then
+      write (line, fmttmb) nhours, nmins, nrsecs
+    elseif (nmins > 0) then
+      write (line, fmttmc) nmins, nsecs, msecs
+    else
+      write (line, fmttmd) nsecs, msecs
+    end if
     call write_message(line, skipafter=1)
-!
-!     Write times to file if requested
-    IF (IPRTIM .GT. 0) THEN
-      IF (NDAYS .GT. 0) THEN
-        WRITE (IOUT, 1010) NDAYS, NHOURS, NMINS, NRSECS
-      ELSEIF (NHOURS .GT. 0) THEN
-        WRITE (IOUT, 1020) NHOURS, NMINS, NRSECS
-      ELSEIF (NMINS .GT. 0) THEN
-        WRITE (IOUT, 1030) NMINS, NSECS, MSECS
+    !
+    ! -- Write times to file if requested
+    if (iprtim > 0) then
+      IF (NDAYS > 0) THEN
+        WRITE (IOUT, fmttma) NDAYS, NHOURS, NMINS, NRSECS
+      ELSEIF (NHOURS > 0) THEN
+        WRITE (IOUT, fmttmb) NHOURS, NMINS, NRSECS
+      ELSEIF (NMINS > 0) THEN
+        WRITE (IOUT, fmttmc) NMINS, NSECS, MSECS
       ELSE
-        WRITE (IOUT, 1040) NSECS, MSECS
+        WRITE (IOUT, fmttmd) NSECS, MSECS
       END IF
     END IF
-!
-    RETURN
-  END SUBROUTINE elapsed_time
+    !
+    ! -- Return
+    return
+  end subroutine elapsed_time
 
-!
-!-------TIMER FOR SUBROUTINES
-  SUBROUTINE code_timer(it, t1, ts)
-! ******************************************************************************
-!     Get end time and calculate elapsed time
-! ******************************************************************************
-!
-!        SPECIFICATIONS:
-! ------------------------------------------------------------------------------
+  !> @brief Get end time and calculate elapsed time
+  !!
+  !! Timer for subroutines
+  !<
+  subroutine code_timer(it, t1, ts)
     ! -- dummy
-    INTEGER(I4B), INTENT(IN) :: it
-    REAL(DP), INTENT(INOUT) :: t1
-    REAL(DP), INTENT(INOUT) :: ts
+    integer(I4B), intent(in) :: it
+    real(DP), intent(inout) :: t1
+    real(DP), intent(inout) :: ts
     ! -- local
-    REAL(DP) :: dt
-! ------------------------------------------------------------------------------
+    real(DP) :: dt
     !
-    IF (IT == 0) THEN
-      call CPU_TIME(t1)
-    ELSE
-      call CPU_TIME(dt)
+    if (it == 0) then
+      call cpu_time(t1)
+    else
+      call cpu_time(dt)
```

Comment:
@langevin-usgs what should the convention be for fortran intrinsics? It might be good use CAPS like we use for parameters (DZERO, etc.).

Reply:
I deliberated on this one, so happy to restore.  See [635cdb7](https://github.com/MODFLOW-USGS/modflow6/pull/1600/commits/635cdb7db296cab37055bd614ec3b9fad0e16652)

Reply:
I don't have a preference.  Will go with whatever is in here now.  Ultimately hoping for a linter to handle this.